<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>version_number</class>
 <widget class="QWidget" name="version_number">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>720</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>AIOC12 v1.1.5_250612</string>
   </property>
  </widget>
  <widget class="Label_version_return" name="label_return">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_backdrop">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>720</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>720</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>当前版本信息</string>
   </property>
  </widget>
  <widget class="Label_version_update" name="label_update">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>升级</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_backdrop</zorder>
  <zorder>label_num</zorder>
  <zorder>label_return</zorder>
  <zorder>label_version</zorder>
  <zorder>label_update</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_version_return</class>
   <extends>QLabel</extends>
   <header>version_number.h</header>
  </customwidget>
  <customwidget>
   <class>Label_version_update</class>
   <extends>QLabel</extends>
   <header>version_number.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
